### PR TITLE
EOS-23243 fix for confstore create failed intermittent issue

### PR DESCRIPTION
**Pull request by zahidsk**
Signed-off-by: zahid.shaikh <zahid.shaikh@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by nitin-seagate — CHANGES_REQUESTED** [review]
Lets change file name:
srv/components/provisioner/confstore_create.sls

File: lr-cli/cortx_setup/commands/pillar_sync.py [hunk 1]
```diff
@@ -51,8 +51,11 @@ def run(self, **kwargs):
         conf_path = str(PRVSNR_FACTORY_PROFILE_DIR / 'confstore')
         # backup local consftore data
         self.logger.debug(f"Copy local confstore file to {conf_path}")
-        state = 'components.provisioner.confstore'
-        StatesApplier.apply([state])
+        state_create = 'components.provisioner.confstore_dir_create'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/pillar_sync.py:54]: Lets change this to:
`conf_create = 'components.provisioner.confstore_create'`

File: lr-cli/cortx_setup/commands/pillar_sync.py [hunk 2]
```diff
@@ -51,8 +51,11 @@ def run(self, **kwargs):
...
+        StatesApplier.apply([state_create], targets=local_minion_id())
+
+        state_copy = 'components.provisioner.confstore_copy'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/pillar_sync.py:57]: Lets change this to:
`conf_copy = 'components.provisioner.confstore_copy'`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/pillar_sync.py (hunk 1 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/pillar_sync.py]: changed

File: lr-cli/cortx_setup/commands/pillar_sync.py (hunk 2 above)
Comment (zahidsk) [reply lr-cli/cortx_setup/commands/pillar_sync.py]: changed var name

**Review by sradhanand-pati — APPROVED** [review]

**Review by 83bhp — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**